app/src/main/java/org/jellyfin/androidtv/ui/playback/AudioNowPlayingFragment.java

```diff
@@ -273,7 +273,8 @@ public void onPlaybackStateChange(@NonNull PlaybackController.PlaybackState newS
 
         @Override
         public void onProgress(long pos) {
-            setCurrentTime(pos);
+            // Round the current time as otherwise the time played and time remaining will not be in sync
+            setCurrentTime(Math.round(pos / 1000) * 1000);
```

Comment:
It would be better to move the rounding to inside the setCurrentTime function to avoid having the same issue if the function were called from some other place